drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -22,7 +22,12 @@ config ETH_SAM_GMAC_NAME
 config ETH_SAM_GMAC_QUEUES
 	int "Number of hardware TX and RX queues"
```

Comment:
This Kconfig symbol specifies the "Number of **active** hardware TX and RX queues".

The maximum number of hardware TX and RX queues supported should be specified in the device tree.